addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -243,54 +243,86 @@
     </t>
 </template>
 
+<template id="course_content_training_slide" name="Slide template for a traning channel">
+    <li t-att-index="j" t-attf-slide_id="#{slide.id}" t-attf-category_id="#{category.id if category else None}" class="content-slide slide-draggable d-flex justify-content-between align-items-center p-2">
+        <div class="content-slide-infos ml-2">
+            <i t-if="channel.can_publish" class="fa fa-arrows mr-2 text-muted"></i>
+            <i t-if="slide.slide_type == 'document'" class="fa fa-file-pdf-o mr-2 text-muted"></i>
```

Comment:
A small tool template slide_icon ?

Reply:
I did that AND I overrode it correctly in website_slides_survey to handle 'certification', call me super-modular-men